orca-core/src/main/java/com/netflix/spinnaker/orca/config/UserConfiguredUrlRestrictions.java

```diff
@@ -81,21 +83,36 @@ public Builder withRejectLinkLocal(boolean rejectLinkLocal) {
       return this;
     }
 
+    public List<String> getRejectedIps() {
```

Comment:
Seems like a lot of boilerplate code can be removed by using lombok `@Data` etc here.